.wordlist.txt

```diff
@@ -0,0 +1,17 @@
+cryptocurrency
```

Comment:
Sorry for being such a noob, but what is this file for?

Reply:
The spellchecker action has a limited vocabulary and flags many false positives. `.wordlist.txt` enriches its vocabulary.

Reply:
Then I don't see this approach scaling in the future. Maybe we have some sort of way of flagging safe words on site the same way as [@trusted](https://dlang.org/spec/function.html#trusted-functions) works in D.

Reply:
Not sure about this. Most of the core vocabulary of the Methodology content is already here and in the long term changes to this file will occur less frequently. On the contrary, flagging words in place in documents leads to a lot of repetition. For example, almost every file would need to flag "Markdown". In addition, the spellchecker's documentation also suggests this approach and not any other, so yours might not be possible: https://github.com/rojopolis/spellcheck-github-actions/tree/master#checking-for-bad-spelling.

Reply:
@razvand , @teodutu , I've opened a discussion on Security Summer School related to the `actions` repository[1], that we use. In short, the ideal would be to have a wordlist file with common words we want to add, inside the `actions` repository[1]. And this `.wordlist.txt` file will only be used for words not already there (which should be very little).

More than that, the `actions` repository, should be part of the `open-education-hub` organization on GitHub. @gabrielmocanu , what do you think about that?

[1] https://github.com/security-summer-school/actions